### PR TITLE
fix(popup-menu): do not search by entry id

### PR DESCRIPTION
**Pull request by smbea**
Related to https://github.com/camunda/camunda-modeler/issues/3347

### CODE REVIEW

File: lib/features/popup-menu/PopupMenuComponent.js [hunk 1]
```diff
@@ -82,10 +82,8 @@ export default function PopupMenuComponent(props) {
       }
 
       const search = [
-        entry.name,
```
[the remark this answers is not in the document]
Comment (nikku) [reply lib/features/popup-menu/PopupMenuComponent.js:85]: Good catch. It should be `entry.label` I believe.

File: test/spec/features/popup-menu/PopupMenuComponentSpec.js [hunk 2]
```diff
@@ -510,6 +510,45 @@ describe('features/popup-menu - <PopupMenu>', function() {
...
+    });
+
+
+    it('should not search id', async function() {
+
+      // given
+      createPopupMenu({ container, entries, search: true });
+
+      var searchInput = domQuery('.djs-popup-search input', container);
+      searchInput.value = '6';
```
Comment (nikku) [inline test/spec/features/popup-menu/PopupMenuComponentSpec.js:539]: Kinda cryptic, I stubmled over this. I'd rather call `6` `JUST_AN_ID` and set the value to that.

File: lib/features/popup-menu/PopupMenuComponent.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nikku) [reply lib/features/popup-menu/PopupMenuComponent.js:85]: (and we already cover `entry.label`) :)

**Review by nikku — COMMENTED** [review]
Please consider a small improvement to the new test: https://github.com/bpmn-io/diagram-js/pull/720/files#r1043262244.

But other than that, great.

File: test/spec/features/popup-menu/PopupMenuComponentSpec.js [hunk 3]
```diff
@@ -440,12 +440,12 @@ describe('features/popup-menu - <PopupMenu>', function() {
...
 
     const entries = [
-      { id: '1', label: 'Entry 1' },
+      { id: '1', label: 'Entry 1', description: 'Entry 1 description' },
       { id: '2', label: 'Entry 2' },
       { id: '3', label: 'Entry 3' },
       { id: '4', label: 'Entry 4' },
       { id: '5', label: 'Entry 5' },
-      { id: '6', label: 'Last' }
+      { id: 'some_entry_id', label: 'Last' }
```
[the remark this answers is not in the document]
Comment (barmac) [reply test/spec/features/popup-menu/PopupMenuComponentSpec.js:448]: This 👍 

Alternatively, we could use entry directly: `input.value = entry[6].id`. It's OK as is though.

**Review by nikku — APPROVED** [review]

**Pull request merged**